pokedex/data/csv/location_areas.csv

```diff
@@ -650,3 +650,26 @@ id,location_id,game_index,identifier
 758,676,0,unknown-area-348
 759,677,0,
 760,661,0,
+761,231,0,museum-of-science
+762,120,0,pokemon-center
+763,234,0,fighting-dojo
+764,67,0,celadon-mansion
+765,234,0,silph-co-7f
+766,229,0,north-gate
+767,229,0,
+768,569,0,
+769,467,0,weather-center
+770,432,0,stevens-house
+771,567,0,
+772,169,0,
+773,170,0,
+774,346,0,
+775,535,0,
+776,349,0,
+777,536,0,
+778,590,0,
+779,599,0,
+780,625,0,
+781,200,0,
+782,578,0,
+783,691,0,
```

Comment:
All these new location areas will need location_area_prose entries.

Reply:
Looking at location_area_prose.csv, it looks like it would be appropriate not to give them names when the corresponding location_area does not have an identifier?

Reply:
Right — if it's the "default area" for a location then the name and identifier should be blank.